lib/index.js

```diff
@@ -8,7 +8,7 @@ var globparent = require('glob-parent');
 
 var NodeFsHandler = require('./nodefs-handler');
 var FsEventsHandler = require('./fsevents-handler');
-exports.isBinaryPath = require('./is-binary');
+exports.isBinaryPath = require('is-binary-path');
```

Comment:
Legacy. It's just been there forever and I've avoided any breaking API changes since I took over. But AFAICT nobody uses it, so now would be the time to remove it from here.